src/m365/pp/commands/dataverse/dataverse-table-list.spec.ts

```diff
@@ -0,0 +1,288 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli';
+import Command, { CommandError } from '../../../../Command';
+
+import request from '../../../../request';
+import { sinonUtil } from '../../../../utils';
+import commands from '../../commands';
+const command: Command = require('./dataverse-table-list');
+
+const envResponse: any = { "properties": { "linkedEnvironmentMetadata": { "instanceApiUrl": "https://contoso-dev.api.crm4.dynamics.com" } } };
```

Comment:
@martinlingstuyl happy to do so, but I did not encounter that anywhere else. Shall we update it in our guidelines as well if we want to make it a part of our default approach?

Reply:
We use multiple approaches:
- Add the mock responses within the test cases if they are only relevant in one test case.
- Add the mock responses as separate consts (like I advised) if they are relevant to multiple test cases.
- Add the mock responses in a separate mock file that you require in the spec file. This is especially relevant when the mocks are very big and the spec file would become unreadable because of it.

We should add it somewhere in the contribution docs. I'm not sure where though.

Reply:
Ok, not a discussion for this one. Lets resolve this one as I implemented it and think 'bout how we can put it in writing in another issue.